### DISCUSSION TITLE
DEM:All grids must be of same extent and resolution

### DISCUSSION THREAD

**Discussion by hhh321747** [UMEP-dev/UMEP#644] [Q&A]
For DEM data and DSM data, I set the same resolution 0.5, the same coordinate system, and the range is also cropped according to a shp plane. Why do errors occur when running the model
![4bb1eec4c8fff12c57fc3fb2dc845c8](https://github.com/user-attachments/assets/abbf79a5-c41d-4a7b-baa4-deb63f0eb151)

**Comment by nilswallenberg** [comment]
Hi,

Can you please check if your DSM and DEM matches in number of pixels at x and y?

Cheers,
Nils

**Comment by hhh321747** [comment]
Yeah, I did the transformation. It's the same thing
![4ba4133888b08332f3bab75a679e1c6](https://github.com/user-attachments/assets/78180631-9e1a-41d6-b0de-efd036a41d1b)
![3581c2aee013eec331b2bf14eb399ef](https://github.com/user-attachments/assets/a5018e03-8a2f-40f8-91e4-7e532d09c128)
![46a46e8aa22f370a8f56f2810b5e453](https://github.com/user-attachments/assets/2eaa619b-93d6-49f9-ba8a-7f5b75d2e022)
![52803c8c04e20dc8ce8b74872c55872](https://github.com/user-attachments/assets/2b326f5f-665c-479a-99a2-edf078117fdb)

**Comment by suegrimmond** [comment]
please can we have key in English

**Comment by nilswallenberg** [comment]
In one of your rasters you have 1516 columns and 839 rows, whereas in the other one you have 1517 and 838. This is why you get an error. They both need to be the same.

Cheers,
Nils

**Reply by hhh321747** [reply]
thank you！
